apps/aecore/src/aec_chain.erl

```diff
@@ -281,6 +284,36 @@ force_insert_headers(HeaderChain) ->
     gen_server:call(?SERVER, {force_insert_headers, HeaderChain},
                     ?DEFAULT_CALL_TIMEOUT).
 
+export_chain(OutFile) ->
+    Headers = all_headers(),
+    Blocks = all_blocks(Headers),
+    file:write_file(
+      OutFile, term_to_binary({{0,1}, Headers, Blocks}, [compressed])).
+
+import_chain(File) ->
+    case file:read_file(File) of
+        {ok, Bin} ->
+            try binary_to_term(Bin) of
+                {{0,1}, Headers, Blocks} ->
+                    io:fwrite("Headers = ~p~n"
+                              "Blocks = ~p~n",
+                              [Headers, Blocks]),
+                    lists:foreach(
+                      fun(H) ->
+                              insert_header(H)
```

Comment:
May you please elaborate on why no match to `ok`? I see the the export make such that `Headers` is from genesis to top.

Reply:
The import caused a crash trying to deserialize transactions, so pending a fix of that, it doesn't work.